### PR TITLE
feat(core): extend authz table with audit attributes

### PR DESCRIPTION
**Pull request by kofzera**
Added created_at, created_by, and created_by_uid columns to authz table and modified the inserts to include them. The setRole method modified to use prepared statement since user supplied input will be in the statement.

### CODE REVIEW

**Review by xflord — CHANGES_REQUESTED** [review]
We treat changes to the db as breaking changes and mark the commits accordingly. Please do so here as well.

File: perun-base/src/test/resources/test-schema.sql [hunk 1]
```diff
@@ -1600,6 +1600,8 @@ create table authz (
 						modified_by_uid integer,
 						authorized_group_id integer, --identifier of whole authorized group
 						security_team_id integer,	--identifier of security team
+						created_at timestamp default statement_timestamp() not null,
+						created_by varchar default user not null,
```
[the remark this answers is not in the document]
Comment (kofzera) [reply perun-base/src/test/resources/test-schema.sql:1604]: > please also add `modified_by` and `modified_at` to match the audit columns in other tables. Same for the `postgres.sql` file. Do not forget to add these to the query statements

At the initial consultation of the task, it was decided that these columns are not needed as there are no methods that update the table.

[the remark this answers is not in the document]
Comment (xflord) [reply perun-base/src/test/resources/test-schema.sql:1604]: I was not aware, it is fine then

**Review by xflord — APPROVED** [review]
Looks good to me

**Review by mattjoke — APPROVED** [review]
Looks good :+1:

**Pull request merged**